abstraignore (#2)
abstra-app/abstra-cli: README.md

``` sh
abstra-cli upload DIRECTORY
```

### Ignoring files

> you can ignore files placing a `.abstraignore` at the beginning of the filea:

Reviewer: ```suggestion
You can ignore files placing a `.abstraignore` at the directory that will be uploaded:
```
